packages/zero-client/src/client/custom.ts

```diff
@@ -74,7 +76,7 @@ export type MakeCustomMutatorInterface<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   F,
 > = F extends (tx: ClientTransaction<S>, ...args: infer Args) => Promise<void>
```

Comment:
TODO: Parameterize the return type too. Replicache allows return values from mutators.